dbt_semantic_interfaces/validations/measures.py

```diff
@@ -37,6 +38,7 @@ class SemanticModelMeasuresUniqueRule(SemanticManifestValidationRule[SemanticMan
     def validate_manifest(semantic_manifest: SemanticManifestT) -> Sequence[ValidationIssue]:  # noqa: D
         issues: List[ValidationIssue] = []
 
+        metrics_by_name = {metric.name: metric for metric in getattr(semantic_manifest, "metrics", [])}
```

Comment:
what's the reason for using `getattr` here instead of accessing the property directly? seems like we get better type safety if we access the property?